### DISCUSSION TITLE
The game does not appreciate persistent players

### DISCUSSION THREAD

**Discussion by IvanSkazkin** [sunflower-land/sunflower-land#2289] [General]
I don't understand. Why were there complex rewards in the great war, but now they are worth nothing?
I became a warrior (got a badge). They said in addition to the fact that there will be access a week earlier, they are preparing something for us.
The warrior helmet was super expensive. They don't cost anything. They say he has a small amount. They produce tiger clothes where there are only 20 pieces.

Is this some kind of joke? Veterans in the game are not appreciated at all?
Well you don't give OG nft boosts. But you yourself said that you love the story in the game.

We fought in the war, got the rarest helmet (armor). And they do nothing at all. And besides, they are not visible under clothes (if only they had a beautiful animation, like wings) for highlighting. But the helmet doesn't look very nice. In chats with OG warriors (which are almost all whales). Everyone has a sense of deprivation. A couple of people even want to leave the game, because the strongest warriors were not rewarded properly.

In general, veterans in this game do not like.
It also harms you. Next time, big players will think many times whether it is worth participating in big events.

You can play as much as you like, this game will only be for making money. There is no merit in front of stubborn players. Alas

**Comment by LandLadyFarmer** [comment]
I don't know how i should feel or say actually about this!! I worked hard through the war and minted full set of armor and got my warrior badge but i don't have any feeling of not being rewarded or cheated of anything...
warrior set was something where there might be something extra and that "might be" was vague so that was nothing you could take serious at all and then Steves talk was something you never could trust....
I'm not felling cheated of anything against how much i put in and if you do that's your opinion...
In how i read your text it seams that you want to profit more from the game rather than have rare stuff for your bumpkin
or just be happy with lower powered boost and i don't say you think is wrong about it just that is something you feel..

As how i feel about everything is that rewarding older players like me or even bigger OG players can wait and shouldn't be prioritized as it is now, most important thing is to have an engaging and healthy on boarding for new players for without them there can't be any more powerful boost we would drain our self out until there was new value...
So until we have an stable on boarding with an steady growing player base and economic it should stay as it is now...

we and i have powerful boost so we will get by some more time and start to help out more as at least be part of sharing ideas, feedback in proposals or whatever you can help in so it will be great for new players to join..
OG players or any beta player needs to set themselves to the side for an while so new players can come then can we others start to demand more and should get more but for now it's new players the focus has to be on and should be not feeding cranky beta players.. I'm sure every OG or whale beta player will be rewarded in one way or another but for now it's not something that should be prioritized...  Thats how i feel and i'm just fine with my full warrior set not having any boost...

Everyone should keep in mind that it is most likely you only hear from those that are unhappy because they have a drive to express theme self others don't so you can never say that everyone is unhappy with whats going on...

**Reply by IvanSkazkin** [reply]
not everyone agrees with you, I read telegram channels and quite a lot of disappointed people. They just have a lot of money and they are too lazy to write. And it's easier to leave the project than to prove something

**Reply by alphacruze** [reply]
You guys are only disappointed because you treat SFL like a financial product not a game

In terms of use, people who got full warrior set could very well have a massive head start when the long await "rock monsters"

And things like that come into play. However in terms of lore, what do you expect 'warrior" items to do in terms of farming?
chop down trees faster? Plant more crops? that's just dumb

A lot of people spend 1000s if not 10's of thousands on stupid skins with nothin to show for it in games, and yet you desire some kind of super powerful buff when all you really had to do was sacrifice excess crops. That was what the war was about, SACRIFICING excess crops.

not producing more to burn them

**Reply by VitorMaxtech** [reply]
Let's be honest, right?! It is a game, ALSO a financial product, once you have to manage resources to be able to survive the game.
What devs want to say with "non financial product" is that SFL team focus on having a long term healthy economy. That can only works if no one is happy about it IMAO... We all as players should lack on something so the economy thrives...
Thing is that a great portion of the players are having a hostage feeling. Cos they are PRETTY disapointed every expension that they should get a REAL UPGRADE, instead they keep getting minimum resourses expansion after expansion. And this is a pain in......
 Game should adopt a point system that evens all the players with diferent resourses wich levels everyone's experience by the path they choose. Like: some farms may have a lot of plots but not much minerals and vice versa.
 I also think that game may launch SFT that not only boosts you but also takes something from you, like: you get 15%more wood but your trees take 8% more to grow.
 I say that because IMAO game must have GAPS provided by OWNING (since its a play to own) NFT/SFTs. This should make economo thrives, not fear to even everyone and break the economy....

**Comment by future2back** [comment]
Where is set buff?

**Reply by alphacruze** [reply]
What kinda of buff you want from warrior?

faster crop growth? lmao. from the get go, warrior set was always just for show, same like a lot of the bumpkin items that's sold on Bumpkins.io

**Comment by alphacruze** [comment]
You're hilarious.

Last I checked a lot of OG Whales have stuff like chicken coop
Carrot Sword
T3 Scarecrow
T3 Beaver
Rock Golem

Hmm... and many many other buffs that New players will never get or they would have to pay hundreds of dollars
Then there's also the fact that most OG's have at 4 to 6 of the legacy skills, another huge advantage over new players

Some OG's got those broken wings...

It's like you guys want everything for free with zero time till harvest or you not happen... ah wait some of them do have that

**Reply by IvanSkazkin** [reply]
It's funny, I'm not saying that I should be given bonuses just like that. I say that there was a very large event, where there were super rare things. For example, a warrior's helmet cost the most and there are only 25 pieces. And it costs less than the green amulet, which was very easy to obtain. Because of the illogicality of this post and left. Even the shield has a bonus, free sunflowers. The helmet is the most expensive, the most valuable. And he has nothing. And it doesn't even have any animation to stand out. They would add at least a fiery overflow to it, at least something. And now this helmet is not even visible in the Kigurumi. Which makes this helmet the most useless, most expensive and most difficult item in the game.

**Comment by 1176517584** [comment]
That's what I was thinking, the skull helmet took too much of the player's passion and ended up being a garbage item. Burning huge resources and financial resources, why not give him a decent ability, only consider the new players do not consider the old players, pay is not proportional to the return, for a long time to go down the loss of more and more players, why Christmas tree a long time ago items, many players do not play, why also airdrop wings to not play people, address has been inactive for more than a year, As a result, few wings are in circulation and prices remain high.

**Reply by IvanSkazkin** [reply]
There are only 25 helmets in the game. Wings 50 pcs. I don't think the economy would break if the helmet was given a buff.

**Reply by TheDcol** [reply]
It was explicitly stated that the war skull was intended to be a pure decoration item for bumpkins

**Comment by DemonDG** [comment]
In addition to the legendary item green necklace, I also think many items in the war version did not perform well. I also saw a discussion on the chat channel about the gameplay of collecting a warrior suit, so I bought a pair of war pants in order to collect a suit. But after all this time, little is known about the moon invaders.

**Comment by DemonDG** [comment]
The Skull Hat is the rarest item in the War version, but so far, it's useless for anything other than total rarity, and we can't even see it in the game.It really became useless.Hope the team takes this into account and empowers war-related parts in subsequent versions of Moon Invaders.

**Comment by adamhannigan** [comment]
Hey @IvanSkazkin Thanks for your feedback!

It's a bit difficult to understand which aspect of the game you are talking about. Is it specifically the Warrior Set, War Items (e.g. Green Amulet, Undead Rooster) or past OG items (Foreman Beaver, Kuebiko etc.)?

Regarding the full warrior set the team believes giving set boosts is a great idea - as we've mentioned lots of times.

However, right now there isn't any buff that fits the cohesion of the game. We can release a War Bond buff, but that would be useless at the moment since there is no war happening. Instead of giving it a useless buff now, we would prefer to wait for a time when it makes sense and gives a useful buff.

Regarding the War Decorations, we were specific at the time which ones provided immediate buffs (e.g Sunflower Shield). The Skull Hat never implied any sort of buff, just a rare cosmetic.

We are open to suggestions to how the set can apply a buff in a way that relates to a warrior outfit 👍

**Reply by DemonDG** [reply]
The Skull Hat from its creation to its current status as a symbol of the strongest warrior, has so far been ignored that we can't even see it in games.It was placed in the corner along with more and more other ornaments.I think it's forgotten.

**Reply by IvanSkazkin** [reply]
> Привет@IvanSkazkinСпасибо за ваш отзыв!
> 
> Немного сложно понять, о каком аспекте игры вы говорите. Это конкретно набор воина, военные предметы (например, зеленый амулет, нежить-петух) или предметы прошлых OG (бригадир бобр, куебико и т. д.)?
> 
> Что касается полного набора воинов, команда считает, что усиление наборов — отличная идея, как мы неоднократно упоминали.
> 
> Тем не менее, сейчас нет ни одного баффа, который соответствовал бы сплоченности игры. Мы можем активировать бафф War Bond, но в данный момент это было бы бесполезно, так как войны не происходит. Вместо того, чтобы давать бесполезный бафф сейчас, мы бы предпочли подождать, пока это будет иметь смысл и даст полезный бафф.
> 
> Что касается боевых наград, то в то время мы четко указывали, какие из них давали немедленные положительные эффекты (например, «Щит подсолнуха»). Шляпа-череп никогда не подразумевала какого-либо баффа, просто редкое косметическое средство.
> 
> Мы открыты для предложений о том, как сет может применять положительный эффект таким образом, чтобы он относился к снаряжению воина.👍

I meant war awards. I understand that even earlier items are essentially just old and they were received for normal game actions at that moment.
But with a set, a helmet and things for the event, it's different. Why? I think that it was a great battle and people here really fought to get these awards, but they unfortunately do not have any bonuses.
You say the helmet is rare. There are 25 of them in the game. And then you release tiger clothes where there are only 20 of them. It turns out that all the rarity of the helmet disappears. It's easier to buy a rare thing than to participate in a war for like 2-3 months and get a thing earned with blood and sweat. And many times more expensive. Also, these clothes cover the helmet.
I don't see the benefits of this great award at all. I always believed that the rewards of the strongest should be somehow rewarded.

I can’t say what the helmet should specifically give. You are a smart person, you have a strong team and I think you understand that it is not logical not to give an award to people for the heaviest award.

You probably know what a helmet can give, you have an economy that you follow. But there are only 25 helmets in the game. To give any bonus for us, for those who played a lot and invested a lot not only money, but also game forces. 

Of course you asked, so here's my suggestion. You can give x2 to the auction ticket (solar flare).  x2 bonus to leave for other seasons, make the bonus permanent. The helmet would show that the most brutal wars can receive seasonal rewards before others. We deserve. The helmet is also a blacksmith's helmet. It probably goes hand in hand with the game's lore. We would be able to receive easier rewards from future mints of the game.
I hope I didn't bore you, sorry. But one more time. In any game, the hardest efforts bring the biggest rewards, and it's not about money, it's about time. Thank you

**Reply by NKbS4s38V** [reply]
> > Привет@IvanSkazkinСпасибо за ваш отзыв!
> > Немного сложно понять, о каком аспекте игры вы говорите. Это конкретно набор воина, военные предметы (например, зеленый амулет, нежить-петух) или предметы прошлых OG (бригадир бобр, куебико и т. д.)?
> > Что касается полного набора воинов, команда считает, что усиление наборов — отличная идея, как мы неоднократно упоминали.
> > Тем не менее, сейчас нет ни одного баффа, который соответствовал бы сплоченности игры. Мы можем активировать бафф War Bond, но в данный момент это было бы бесполезно, так как войны не происходит. Вместо того, чтобы давать бесполезный бафф сейчас, мы бы предпочли подождать, пока это будет иметь смысл и даст полезный бафф.
> > Что касается боевых наград, то в то время мы четко указывали, какие из них давали немедленные положительные эффекты (например, «Щит подсолнуха»). Шляпа-череп никогда не подразумевала какого-либо баффа, просто редкое косметическое средство.
> > Мы открыты для предложений о том, как сет может применять положительный эффект таким образом, чтобы он относился к снаряжению воина.👍
> 
> I meant war awards. I understand that even earlier items are essentially just old and they were received for normal game actions at that moment. But with a set, a helmet and things for the event, it's different. Why? I think that it was a great battle and people here really fought to get these awards, but they unfortunately do not have any bonuses. You say the helmet is rare. There are 25 of them in the game. And then you release tiger clothes where there are only 20 of them. It turns out that all the rarity of the helmet disappears. It's easier to buy a rare thing than to participate in a war for like 2-3 months and get a thing earned with blood and sweat. And many times more expensive. Also, these clothes cover the helmet. I don't see the benefits of this great award at all. I always believed that the rewards of the strongest should be somehow rewarded.
> 
> I can’t say what the helmet should specifically give. You are a smart person, you have a strong team and I think you understand that it is not logical not to give an award to people for the heaviest award.
> 
> You probably know what a helmet can give, you have an economy that you follow. But there are only 25 helmets in the game. To give any bonus for us, for those who played a lot and invested a lot not only money, but also game forces.
> 
> Of course you asked, so here's my suggestion. You can give x2 to the auction ticket (solar flare). x2 bonus to leave for other seasons, make the bonus permanent. The helmet would show that the most brutal wars can receive seasonal rewards before others. We deserve. The helmet is also a blacksmith's helmet. It probably goes hand in hand with the game's lore. We would be able to receive easier rewards from future mints of the game. I hope I didn't bore you, sorry. But one more time. In any game, the hardest efforts bring the biggest rewards, and it's not about money, it's about time.. Thank you

Totally agree. Skull Hat was much harder to get than green amulet, the emission is 32 times less, but now it costs nothing, no abilities. Everyone who was active in participating in war burnt lots of resources and got nothing. Instead of usefull nft's in war tent only useless decorations were released. Decorative nft's that cost nothing, chance to exchange warbonds for nothing and warrior badge that gave me and some other players opportunity to get early access, buy sand shovels for x5 price and again lose lots of resources without any compensations. A year ago I started to play a game that was rewarding those who play hard and a lot. Right now only lucky or those who have insider info can be successful. I, as lots of other players, sold all my starfish's just before the iron boost nft release, I had no idea that beach bounty may be used as a resource for mints and now there is almost no chance that I can get this extremely useful nft just by playing. Why not just release it at the same time with treasure island? Luckily half of people minted potato boost nft for 0, and now sell them for 20 bucks, and half spent their resources. It's just fortune.

**Reply by Nikollini** [reply]
> Hey @IvanSkazkin Thanks for your feedback!
> 
> It's a bit difficult to understand which aspect of the game you are talking about. Is it specifically the Warrior Set, War Items (e.g. Green Amulet, Undead Rooster) or past OG items (Foreman Beaver, Kuebiko etc.)?
> 
> Regarding the full warrior set the team believes giving set boosts is a great idea - as we've mentioned lots of times.
> 
> However, right now there isn't any buff that fits the cohesion of the game. We can release a War Bond buff, but that would be useless at the moment since there is no war happening. Instead of giving it a useless buff now, we would prefer to wait for a time when it makes sense and gives a useful buff.
> 
> Regarding the War Decorations, we were specific at the time which ones provided immediate buffs (e.g Sunflower Shield). The Skull Hat never implied any sort of buff, just a rare cosmetic.
> 
> We are open to suggestions to how the set can apply a buff in a way that relates to a warrior outfit 👍

I agree that I was not rewarded enough. It was very difficult to get a badge of a warrior. And in the end we got nothing. A week early entry, it's NOTHING.
Also, the warrior set was not given any boost, although there were talks about it. Skull Hat is kind of a joke. Gives nothing, was the most expensive. There are some rarer things.
Bottom line, the war was difficult, long, interesting. The end of the event is disappointing. I thought there would be awards and some honors to the most powerful warriors. And why I fought for a badge and a helmet, I don't know

**Reply by 1176517584** [reply]
Adam, I just want to ask you, why is it that everyone else has two starfish every day, I have almost zero starfish every day, I've only had 18 starfish since Treasure Island, I've been digging 50 shovels every day, I've consumed countless resources, and it cost me thousands of dollars during the war, on average 0.01 starfish a day? Others have been after me since 0, is it because I have Kumiko and foreman Beaver deliberately screw me, this is the most angry since playing the game, if this is your so-called fair, then make a new treasure item instead of using this item to forge ！！！！ when others already have a lot of it

**Comment by adamhannigan** [comment]
Some great discussion. Here are some overall notes:

I hope I recall correctly, but the top Warriors received multiple resources (+ some rare items) and a bonus Warrior Badge as a status symbol. As promised, we also gave the Warrior badge holders early access to Land Expansion.

The Skull Hat was only every destined as a cosmetic, the other items with buffs were clearly labelled. However, it should be showing on the NPC inside of the game, thanks for noticing that. We will fix it ✅ 

I can see as a collector you wouldn't want any other rare wearables getting released - this would make your item more unique. However, to drive engagement and grow our player base we will keep releasing exciting content for our players.

We agree with all your comments around sets getting buffs 👍 This has always been on our stance, we just haven't been able to squeeze it onto our roadmap or find a suitable use case for it.

**Reply by IvanSkazkin** [reply]
> Some great discussion. Here are some overall notes:
> 
> I hope I recall correctly, but the top Warriors received multiple resources (+ some rare items) and a bonus Warrior Badge as a status symbol. As promised, we also gave the Warrior badge holders early access to Land Expansion.
> 
> The Skull Hat was only every destined as a cosmetic, the other items with buffs were clearly labelled. However, it should be showing on the NPC inside of the game, thanks for noticing that. We will fix it ✅
> 
> I can see as a collector you wouldn't want any other rare wearables getting released - this would make your item more unique. However, to drive engagement and grow our player base we will keep releasing exciting content for our players.
> 
> We agree with all your comments around sets getting buffs 👍 This has always been on our stance, we just haven't been able to squeeze it onto our roadmap or find a suitable use case for it.

It turns out that for a set of helmet armor and pants you have an idea for buffs, but not for a helmet? Maybe he can become part of the set and also give a bonus, but instead of a helmet and more powerful?

**Reply by NKbS4s38V** [reply]
> > Some great discussion. Here are some overall notes:
> > I hope I recall correctly, but the top Warriors received multiple resources (+ some rare items) and a bonus Warrior Badge as a status symbol. As promised, we also gave the Warrior badge holders early access to Land Expansion.
> > The Skull Hat was only every destined as a cosmetic, the other items with buffs were clearly labelled. However, it should be showing on the NPC inside of the game, thanks for noticing that. We will fix it ✅
> > I can see as a collector you wouldn't want any other rare wearables getting released - this would make your item more unique. However, to drive engagement and grow our player base we will keep releasing exciting content for our players.
> > We agree with all your comments around sets getting buffs 👍 This has always been on our stance, we just haven't been able to squeeze it onto our roadmap or find a suitable use case for it.
> 
> It turns out that for a set of helmet armor and pants you have an idea for buffs, but not for a helmet? Maybe he can become part of the set and also give a bonus, but instead of a helmet and more powerful?

+++
Skull Hat is a big disappointment, minting price was more than 400$ but... As i said before to be succesful now you dont need to play much, you need just to be lucky or to know inside info. Mint some items, dont mint some other, buy green amulet before it gets UNEXPECTABLY buffed from working once every 30-40 harvests to once every ~10, unexpectably mint peeled potato for free, dont sell your starfishes before an UNEXPECTABLE iron idol release to get advantage in a race, withdraw all resources you need before UNEXPECTABLE 30% tax, dont play hard at war because all you will be able to do with your warbonds is mint multiple useless cheap decorations or clothes with no utility, useful nfts wont be released, the main goal is to burn resources that active players have and benefit those who play less, so you will just lose LOTS of resourses. And many other ways to take active player`s resources and give them to unactive. Spend your resources on expanding to get RANDOM combination. From game about farming SFL becomes game about gambling. Will there be early sand shovel crafters compensations? When black chicken decoration?

**Reply by Nikollini** [reply]
> > > Some great discussion. Here are some overall notes:
> > > I hope I recall correctly, but the top Warriors received multiple resources (+ some rare items) and a bonus Warrior Badge as a status symbol. As promised, we also gave the Warrior badge holders early access to Land Expansion.
> > > The Skull Hat was only every destined as a cosmetic, the other items with buffs were clearly labelled. However, it should be showing on the NPC inside of the game, thanks for noticing that. We will fix it ✅
> > > I can see as a collector you wouldn't want any other rare wearables getting released - this would make your item more unique. However, to drive engagement and grow our player base we will keep releasing exciting content for our players.
> > > We agree with all your comments around sets getting buffs 👍 This has always been on our stance, we just haven't been able to squeeze it onto our roadmap or find a suitable use case for it.
> > 
> > 
> > It turns out that for a set of helmet armor and pants you have an idea for buffs, but not for a helmet? Maybe he can become part of the set and also give a bonus, but instead of a helmet and more powerful?
> 
> +++ Skull Hat is a big disappointment, minting price was more than 400$ but... As i said before to be succesful now you dont need to play much, you need just to be lucky or to know inside info. Mint some items, dont mint some other, buy green amulet before it gets UNEXPECTABLY buffed from working once every 30-40 harvests to once every ~10, unexpectably mint peeled potato for free, dont sell your starfishes before an UNEXPECTABLE iron idol release to get advantage in a race, withdraw all resources you need before UNEXPECTABLE 30% tax, dont play hard at war because all you will be able to do with your warbonds is mint multiple useless cheap decorations or clothes with no utility, useful nfts wont be released, the main goal is to burn resources that active players have and benefit those who play less, so you will just lose LOTS of resourses. And many other ways to take active player`s resources and give them to unactive. Spend your resources on expanding to get RANDOM combination. From game about farming SFL becomes game about gambling. Will there be early sand shovel crafters compensations? When black chicken decoration?

I didn't count before, but if so. This is of course very embarrassing. $400 is 4000 sfl. Helmet 25 pieces.
Tiger clothes 20 pieces - 500 sfl. It's rarer, cheaper, and easy to get, just buy it. Yes, I'm in shock

**Comment by BOJack4** [comment]
Hello @adamhannigan Maybe you could just announce in the discord that the warrior helmet will get an update sometime, but it may not be soon, in 1-2 or even 3 years. But it will be, you just don't know when the event will be needed. It would make it very easy to understand. I, as the owner of the golden rooster, am grateful to you that you immediately said in the discord that the egg will be with buffs, and I have been waiting for almost a year. I didn't have any negatives.

I understand the people here, although I myself do not own a helmet and a warrior badge, they want to get guarantees from you that their work will be rewarded, albeit not soon.
Thank you

**Reply by IvanSkazkin** [reply]
I agree, if the team just confirmed that someday he would have a boost, I would be calm. No matter how long, just need an official statement

**Reply by adamhannigan** [reply]
I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.

**Reply by IvanSkazkin** [reply]
> I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.

Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?

**Reply by BOJack4** [reply]
> > I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.
> 
> Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?

Also interested in this question

**Reply by NKbS4s38V** [reply]
> > > I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.
> > 
> > 
> > Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?
> 
> Also interested in this question

+++

**Reply by GramodUFO** [reply]
> > I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.
> 
> Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?

The most important question of this discussion

**Reply by Nikollini** [reply]
> I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.

I agree with everyone, very important question

**Reply by YamiYamiKokiKopi** [reply]
> > I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.
> 
> Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?

++++++++++++ important question

**Reply by GeekBrains72** [reply]
> > I feel we have been consistent with our wording, the team wants to add 'set buffs'. There is just no timeline yet.
> 
> Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?

I hope the team will be able to answer this question

**Comment by adamhannigan** [comment]
Thanks for those that raised Skull Hat was missing in game. The art for this cosmetic was done a long time ago but never added - it will be included in the next release ✅ 

https://github.com/sunflower-land/sunflower-land/pull/2328

**Comment by GeekBrains72** [comment]
@adamhannigan "Adam, could you please answer if The Skull Hat will be included in set or be given an independent buff?"

**Reply by YamiYamiKokiKopi** [reply]
the team cannot probably answer this question
